RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -11,6 +11,7 @@
 #include "vdt/vdtMath.h"
 
 using namespace std;
+#define VI_DEBUG
```

Comment:
Repeating here a comment of mine that got removed while re-indenting everything

VI_DEBUG should probably get un-defined for standard usages

Further: in the code you combine the usage of this macro and the "verbose_" flag: they mostly coincide, but not everywhere. Please review throughout the code and verify that they always correspond to what you want (e.g. check if you want to enclose within #ifdef VI_DEBUG (almost) every appearance of the "if (verbose_)" )